circle.yml

```diff
@@ -29,5 +30,6 @@ deployment:
     owner: palantir
     commands:
       - ./gradlew -i bintrayUpload
+      - ./gradlew -i :gradle-sls-packaging:extractYourkitAgent
```

Comment:
rather than explicitly adding these here, could we just add a few lines to one of our build.gradles so that the task is run automatically?

```
tasks.idea.dependsOn extractYourkitAgent
tasks.compileJava.dependsOn extractYourkitAgent
```